sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -25,13 +25,18 @@
     _SUBSCRIPTION_ID_ENV_VAR.split("+")[0] if _SUBSCRIPTION_ID_ENV_VAR else None
 )
 _logger = logging.getLogger(__name__)
+_logger.propagate = False
+_logger.setLevel(logging.INFO)
 _DIAGNOSTIC_LOG_PATH = _get_log_path()
+_ATTACH_SUCCESS_DISTRO = "4200"
```

Comment:
So I'm assuming we have the 4XXX - 4999 range for python? We should probably document this somewhere right?

Reply:
Yes. I'll edit that line of the [spec](https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/blob/main/ApplicationInsights/applens-detector-app-service-attach.md)

Reply:
https://github.com/aep-health-and-standards/Telemetry-Collection-Spec/pull/206/files